doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -60,12 +62,15 @@ Displaying
    This command displays information about the current state of the
    environment, including sections and modules.
 
-   .. cmdv:: Inspect @num
+   .. cmd:: Inspect @num
```

Comment:
I think related commands should go to the same place, and I'm not bothered if they don't have the same prefixes.  Whether they should be considered "variants" of each other is up for debate.

Reply:
Yes, although this is mostly a comment about good organization of the refman, which we are trying to improve.